USAGE.md

```diff
@@ -27,9 +31,24 @@ export AWS_SECRET_ACCESS_KEY=$(echo $credentials | jq -r ".Credentials[\"SecretA
 export AWS_SESSION_TOKEN=$(echo $credentials | jq -r ".Credentials[\"SessionToken\"]")
 ```
 
-# Data Lake Lambda Endpoints Usage
-## Endpoint Request Format
-```
+## Dataset format
+
```

Comment:
I would add note here what makes valid dataset - geospatial data files + geospatial metadata files (in STAC format)